### ISSUE TITLE
typo in FC layer

### ISSUE THREAD

**Issue by kshmelkov** [ry/tensorflow-resnet#8]
Hello,

In `resnet.py:283` you use `FC_WEIGHT_STDDEV` even for weight decay which looks wrong for me. Probably just a copy-paste bug.

**Comment by achaiah** [comment]
Out of curiosity, what should it be?

**Comment by kshmelkov** [comment]
`FC_WEIGHT_DECAY`. Which already exists, but just isn't used. For equivalent code in conv layers `CONV_WEIGHT_DECAY` is used.
